library/src/main/java/org/xmtp/android/library/messages/PrivateKey.kt

```diff
@@ -36,10 +36,6 @@ class PrivateKeyBuilder : SigningKey {
         privateKey = key
     }
 
-    constructor(encodedPrivateKeyData: String) {
```

Comment:
I removed these in favor of a privateKeyBundleV1 encoder that can be used both in the generate new wallet & connect existing wallet flows: https://github.com/xmtp/xmtp-android/pull/48/files#diff-7c7d7c0b973262f1159dfb738cbef2159416b59800119179a60f65cb705181d6R11